### ISSUE TITLE
[Hold #240916] IOU and Split Bill chats not showing as unread

### ISSUE THREAD

**Issue by puneetlath** [Expensify/App#13913]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Account A and account B have never chatted
2. Account B sends account A a IOU request
3. The chat with the money request shows up in the LHN but is not unread

## Expected Result:
The chat would be unread

## Actual Result:
The chat shows up as read
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [x] Android / Chrome
- [x] iOS / native
- [x] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by puneetlath** [comment]
Gonna put this on hold until we finish the transition to sequence numbers. Then will see if this is still an issue.

**Comment by puneetlath** [comment]
Still on hold. Should be able to work on it soon though.

**Comment by puneetlath** [comment]
Will look into this this week if it is still happening.

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~01580c91187cf88b1f

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor Plus for review of internal employee PR - @parasharrajat (`Internal`)

**Comment by puneetlath** [comment]
Woof, it took me forever to figure out and I was sure the issue was on the API side, but it turns out to be a one-line change on the client 🤦🏾.

**Comment by MelvinBot** [comment]
@puneetlath, @parasharrajat Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by puneetlath** [comment]
Yep, working on a fix.

**Comment by MelvinBot** [comment]
@puneetlath, @parasharrajat Eep! 4 days overdue now. Issues have feelings too...

**Comment by MelvinBot** [comment]
@puneetlath, @parasharrajat 6 days overdue. This is scarier than being forced to listen to Vogon poetry!

**Comment by puneetlath** [comment]
Auth PR is in review. Will be able to ask for a re-review on the App PR once that one is deployed.

**Comment by MelvinBot** [comment]
@puneetlath, @parasharrajat Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by MelvinBot** [comment]
@puneetlath, @parasharrajat Huh... This is 4 days overdue. Who can take care of this?

**Comment by MelvinBot** [comment]
@puneetlath, @parasharrajat Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.88-2 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15052


If no regressions arise, payment will be issued on 2023-03-31. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@parasharrajat / @puneetlath] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@parasharrajat / @puneetlath] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@parasharrajat / @puneetlath] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@puneetlath] Determine if we should create a regression test for this bug.
- [ ] [@parasharrajat] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@puneetlath] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by puneetlath** [comment]
@parasharrajat could you help with the checklist?

**Comment by puneetlath** [comment]
@parasharrajat I've also sent you a contract here: https://www.upwork.com/nx/wm/pre-hire/c/8577561/offer/23878831

**Comment by puneetlath** [comment]
Thanks for the bumps @parasharrajat and sorry for the delay. That all looks good to me. I've gone ahead and paid out the contract. Thank you!
